### DISCUSSION TITLE
How to add power_on_behavior to supported device?

### DISCUSSION THREAD

**Discussion by spry-salt** [Koenkk/zigbee2mqtt#16403] [Devices]
I have a bunch of these:
https://www.zigbee2mqtt.io/devices/GP-WOU019BBDWG.html

They used to support setting the power_on_behavior but somewhere along the way zigbee2mqtt no longer offers that. How would I add that back please?

**Comment by kolcun** [comment]
I'm in a similar spot with these devices https://www.zigbee2mqtt.io/devices/3RSP019BZ.html
The docs indicate the support is there, but it doesn't work.

**Reply by kolcun** [reply]
Firmware upgrading the devices enabled the functionality.
Third Reality Support was helpful in this regard
